src/aws/osml/tile_server/viewpoint/viewpoint_id/__init__.py

```diff
@@ -0,0 +1,12 @@
+#  Copyright 2024 Amazon.com, Inc. or its affiliates.
+
+from .image import (
+    get_image_bounds,
+    get_image_crop,
+    get_image_info,
+    get_image_metadata,
+    get_image_preview,
+    get_image_statistics,
+    get_image_tile,
+)
+from .map import _invert_tile_row_index, get_map_tile, get_map_tileset_metadata, get_map_tilesets
```

Comment:
Looks like _invert_tile_row_index was intended to be a private function. I expect it is only called by get_map_tile. Does it need to be exposed outside of the map package?

Reply:
Cleaned up all the __init__.py files.